charts/images.yaml

```diff
@@ -278,11 +278,11 @@ images:
 - name: istio-proxy
   sourceRepository: github.com/istio/istio
   repository: gcr.io/istio-release/proxyv2
-  tag: "1.10.2-distroless"
+  tag: "1.12.0-distroless"
```

Comment:
Do you plan also to update to go.mod dependency to 1.12.0? When I check previous upgrade PRs, both image version and go.mod dependency version were updated in a single PR - for example https://github.com/gardener/gardener/pull/4241.

Reply:
Thank you for the hint, updated it accordingly.